MobileCenterPush/MobileCenterPush/Internal/Model/MSPushLog.m

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ */
+
+
+#import "MSPushLog.h"
+
+static NSString *const kMSTypePushInstallationType = @"push_installation";
+
+static NSString *const kMSPushInstallationId = @"installation_id";
+static NSString *const kMSPushChannel = @"push_channel";
+static NSString *const kMSPushPlatform = @"platform";
+static NSString *const kMSPushTags = @"tags";
+
+@implementation MSPushLog
+
+@synthesize type = _type;
+
+- (instancetype)init {
+  self = [super init];
+
+  if (self) {
+    _type = kMSTypePushInstallationType;
+    _platform = @"apns";
+  }
+  return self;
+}
+
+- (NSMutableDictionary *)serializeToDictionary {
+  NSMutableDictionary *dict = [super serializeToDictionary];
+
+  if (self.installationId) {
+    dict[kMSPushInstallationId] = self.installationId;
+  }
+  if (self.pushChannel) {
+    dict[kMSPushChannel] = self.pushChannel;
+  }
+
```

Comment:
Inconsistent format. Remove all lines before (or after) `if` statement unless you want to put comments there.